tutorials/units-and-integration/units-and-integration.ipynb

```diff
@@ -0,0 +1,416 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f580cb886a0>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "nu = np.linspace(1., 3000., 1000) * u.THz\n",
+    "bb5000K_nu = blackbody_nu(in_x=nu, temperature=5000. * u.Kelvin)\n",
```

Comment:
Would be great if you can do the same thing with just `Blackbody1D` model, which supports units. Although it can be tricky to get the `bolometric_flux` parameter set right. See astropy/astropy#6456 .